iOS-School-02_stack.playground/Contents.swift

```diff
@@ -0,0 +1,65 @@
+// MARK: - Stack
```

Comment:
Add blank line around `// MARK: -`s

```suggestion
// MARK: - Stack

```

Reply:
Please double check the rest.

Reply:
Fixed.